src/main.js

```diff
@@ -105,22 +97,22 @@ function homeVisibility() {
   saveCoverButton.classList.remove("hidden");
 };
 
+function pushAndSetFieldValues() {
+  covers.push(coverField.value);
+  titles.push(titleField.value);
+  descriptors.push(desc1Field.value, desc2Field.value);
+  coverImg.setAttribute("src", coverField.value);
+  title.innerText = titleField.value;
+  descriptor1.innerText = desc1Field.value;
+  descriptor2.innerText = desc2Field.value;
+};
+
 function createNewBook(event) {
```

Comment:
Try to stick with consistent language when naming functions as well. I've seen both "cover" and "book" used to describe the same thing. Might get a bit confusing